### ISSUE TITLE
Error submitting organization - Country missing problem?

### ISSUE THREAD

**Issue by tamielbr** [hbz/oerworldmap#800]
Got this screen. When I added country (it was empty) it accepted input. Perhaps that is the problem.

![screen shot 2016-07-30 at 11 25 47](https://cloud.githubusercontent.com/assets/3904166/17271070/86256388-5648-11e6-9b1f-8b32b2e5feae.png)

**Comment by literarymachine** [comment]
> When I added country (it was empty) it accepted input. Perhaps that is the problem.

Exactly. Unfortunately this is still quite cryptic, but `"pointer":"/location/address"` and `missing: ["addressCountry"]` is the important part.
